src/gui/symbology-ng/qgssymbollayerv2widget.cpp

```diff
@@ -1588,58 +1617,58 @@ class QgsSvgListModel : public QAbstractListModel
     {
       Q_UNUSED( parent );
       return mSvgFiles.count();
-    }
+  }
```

Comment:
here's where the spacing issue starts... the spacing looks odd from here to the end of the file.